homeassistant/components/media_player/sonos.py

```diff
@@ -351,11 +344,16 @@ def play_media(self, media_type, media_id, **kwargs):
             self._player.play_uri(media_id)
 
     @only_if_coordinator
-    def group_players(self):
+    def group_players(self, service):
```

Comment:
That's weird, those can be removed. A device should not be aware of service calls.